build.sbt

```diff
@@ -1,51 +1,45 @@
+import Versions._
 import BuildHelper._
 
+enablePlugins(EcosystemPlugin)
+
 Global / onChangedBuildSource := ReloadOnSourceChanges
 
 inThisBuild(
   List(
-    organization := "dev.zio",
-    homepage     := Some(url("https://zio.dev/zio-cache/")),
-    licenses     := List("Apache-2.0" -> url("http://www.apache.org/licenses/LICENSE-2.0")),
+    organization       := "dev.zio",
+    homepage           := Some(url("https://zio.dev/zio-cache/")),
+    licenses           := List("Apache-2.0" -> url("http://www.apache.org/licenses/LICENSE-2.0")),
```

Comment:
licenses, homepage, and organization can have defaults?

Reply:
Done!